### PR TITLE
Send a confirmation email upon registering 

### PR DESCRIPTION
**Pull request by qurbat**

### CODE REVIEW

File: funnel/jobs/jobs.py [hunk 1]
```diff
@@ -13,9 +13,12 @@
...
 
 @rq.job('funnel')
-def send_mail_async(sender, to, body, subject):
+def send_mail_async(sender, to, body, subject, file=None):
     with app.app_context():
-        send_mail(sender, to, body, subject)
+        if file is not None:
+            send_mail(sender, to, body, file, subject)
+        else:
+            send_mail(sender, to, body, subject)
```
Comment (iambibhas) [inline funnel/jobs/jobs.py:21]: Check if it's ok to pass `file=None` to `send_mail`, then you dont need this if-else block.

File: funnel/templates/project_registered_email.html.jinja2 [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Hi {{ current_auth.user.fullname.split()[0] }},
```
Comment (iambibhas) [inline funnel/templates/project_registered_email.html.jinja2:1]: It's better if you can add a property called `first_name` in User models and do this there. Or just use fullname.

File: funnel/views/helpers.py [hunk 3]
```diff
@@ -192,10 +192,12 @@ def url_join(base, url=''):
     return urljoin(base, url)
 
 
-def send_mail(sender, to, body, subject):
+def send_mail(sender, to, body, subject, file=None):
     msg = Message(sender=sender, subject=subject, recipients=[to])
     msg.body = body
     msg.html = markdown(msg.body)  # FIXME: This does not include HTML head/body tags
+    if file is not None:
```
Comment (iambibhas) [inline funnel/views/helpers.py:199]: Same comment as the async function.

File: funnel/views/project.py [hunk 4]
```diff
@@ -571,7 +580,10 @@ def deregister(self):
     @requires_login
     @requires_roles({'concierge'})
     def rsvp_list(self):
-        return {'project': self.obj, 'statuses': RSVP_STATUS}
+        return {
+            'project': self.obj,
+            'going_rsvps': self.obj.rsvps_with(RSVP_STATUS.YES),
```
Comment (iambibhas) [inline funnel/views/project.py:585]: pull master in this branch.

File: funnel/views/schedule.py [hunk 5]
```diff
@@ -124,6 +124,24 @@ def schedule_data(project, with_slots=True, scheduled_sessions=None):
     return schedule
 
 
+def schedule_ical(sched):
+    cal = Calendar()
+    cal.add('prodid', "-//HasGeek//NONSGML Funnel//EN")
```
Comment (iambibhas) [inline funnel/views/schedule.py:129]: Do we need to mention Funnel here?

File: funnel/templates/project_registered_email.html.jinja2 (hunk 2 above)
Comment (jace) [reply funnel/templates/project_registered_email.html.jinja2:1]: This should remain `fullname`. Do not assume the first part is the given name.

**Pull request closed without merging**